rules/judgment.tex

```diff
@@ -39,20 +39,36 @@ \subsection{Head Referee}
 \subsection{Assistant Referees}
 \label{sec:assist_referee}
 
-The two assistant referees handle the robots and the ball.
+The assistant referees handle the robots and the ball.
 They take the robots out when they are penalized, and they put the robots in again.
 If a team requests to pick up a robot, an assistant referee will pick it up and give it to one of the team members once the head referee approves.
 An assistant referee will also put the robot back on the field.
 An assistant referee will also replace the ball when it goes off the field or becomes stuck between a players feet.
-
-The head referee should talk to the assistant referees before the game starts to determine what tasks or lesser decisions they wish to delegate to them, however the final calls must come from the head referee.
-Examples of delegated assistance to referee decisions include keeping track of indirect-kick rule compliance to help confirm if a valid goal has been scored and indicating possible rules violations requiring a penalty (such as leaving the field, or motion in set).
+At the discretion of the head referee, more tasks can be delegated to the assistants (\cref{sec:referee_delegation}).
 
 Assistant referees should only enter the field to execute a decision made by the main referee.
 They should not prevent robots from falling during the game.
 
+A game has at least two assistant referees. If agreed upon by the referee teams
+or under certain circumstances, additional assistants may be present,
+up to a total maximum of four. See \cref{sec:refSelection}.
+
 The assistant referees should be equipped with a suitable referee jersey, and black or dark-blue socks.
 
+\subsubsection{The Standing Assistant}
+
+Up to one assistant referee may be designated as the \textit{standing assistant}.
+This is especially encouraged if the game includes additional assistants
+beyond the standard two.
+
+If a \textit{standing assistant} is present, they are tasked with standing
+on the T-junction opposite to the technical area at all times and perform
+the gestures for the beginning of the \texttt{ready} phase and for set plays
+instead of the head referee.
+
+They may be delegated other tasks by the head referee, such as keeping track
```

Comment:
I would be more in favor of strongly encouraging this as well because when the referee is doing the gesture it also knows that from now on the indirect kick counter ist reset.